gallery/src/guidelines/ColorPalette/Swatcher.scss

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+ @import '~@sonatype/react-shared-components/scss-shared/nx-colors';
+ @import '~@sonatype/react-shared-components/scss-shared/nx-variables';
+ @import '~@sonatype/react-shared-components/scss-shared/nx-text-helpers';
+
+.gallery-swatches {
```

Comment:
I wonder if these would be good candidates to make as `.nx-card`s

Reply:
> I wonder if these would be good candidates to make as `.nx-card`s

You know I did think about that, I didn't just because the `NxCard` PR is still in review. We could hold on this until it's merged.

Reply:
> > I wonder if these would be good candidates to make as `.nx-card`s
> 
> You know I did think about that, I didn't just because the `NxCard` PR is still in review. We could hold on this until it's merged.

`.nx-card` implementation. I tweaked the `max-width` a bit so that the longer variables didn't wrap.